specs/testing/SkillsMatrixTesting.md

```diff
@@ -0,0 +1,58 @@
+# Automated Testing of Skills
+Defines the short term goals around the automated testing of skills. 
+
+# Goal
+Automate the testing matrix for Bot/Skill interactions. The full testing matrix runs every night. Failures result in emails. 
+
+# Version of the Botbuilder SDK
+1. For each test run, the latest *preview* Bot Builder SDK should be consumed from the MyGet.
+
+2. Alternatly, a specific version of the BotBuilder SDK could be specified via an Azure Devops variable. 
+
+# Testing Matrix
+The following testing matrix needs to be fully tested
+|   |        |    | **Skill**    |        |   |
```

Comment:
Dotnet has 2 flavors that have different integration libraries. I wonder if there is value in testing .net core vs framework as different elements in the matrix. I will inspect the code to see if there is skill speciifc code in this layer. If there is not, that means that potentially only one c# entry could suffice.

Reply:
Absolutly. I have expacted the doc to include the crawl / walk / run steps. I wanted to start simple and get immediate value this milestone, rather than have a large matrix that we never finish.